### PR TITLE
Remove the graphite handler

### PR DESCRIPTION
**Pull request by somic**
Latest sensu-puppet master now includes /etc/sensu/conf.d/mutators (https://github.com/sensu/sensu-puppet/pull/435 - in manifests/package.pp) which conflicts with our graphite.pp

Pinned puppet-sensu to an earlier version to let tests pass

### CODE REVIEW

**Comment by solarkennedy** [comment]
Instead of pinning can you take the mutators dir out of ours? Heck, I vote we take all the graphite stuff out, we no longer use it.

**Comment by somic** [comment]
Redid this PR by removing graphite handler altogether.

File: manifests/graphite.pp [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-      graphite => {
-        command => 'RUBYOPT=rubygems /usr/share/sensu-community-plugins/mutators/graphite.rb --reverse'
-      }
-    }
-  }
-  file { '/etc/sensu/conf.d/mutators/graphite.json':
-      owner   => root,
-      group   => root,
-      mode    => '0444',
-      content => inline_template('<%= require "json"; JSON.generate @graphite_data %>'),
```
Comment (solarkennedy) [inline manifests/graphite.pp]: Heh. Yuck.

File: README.md [hunk 2]
```diff
@@ -95,10 +95,6 @@ This is a modification of the `ec2_node` community handler. It caches the list
 of ec2 instances from the Amazon API and will automatically remove servers
 from Sensu if they do not exist in the API.
 
-#### graphite
-
-Standard handler, sends graphite metrics.
```
Comment (solarkennedy) [inline README.md]: Yea, the fact that it is a standard handler, to me, hints that it shouldn't really be in this puppet module. This module isn't meant to store and deploy every conceivable sensu handler, just our special ones.

**Comment by solarkennedy** [comment]
Ship!

**Pull request merged**